authorization/v1/0000_03_authorization-openshift_01_rolebindingrestriction.crd.yaml

```diff
@@ -1,31 +1,29 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  name: rolebindingrestrictions.authorization.openshift.io
   annotations:
+    api-approved.openshift.io: https://github.com/openshift/api/pull/470
     include.release.openshift.io/ibm-cloud-managed: "true"
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
+  name: rolebindingrestrictions.authorization.openshift.io
 spec:
   group: authorization.openshift.io
-  scope: Namespaced
   names:
     kind: RoleBindingRestriction
     listKind: RoleBindingRestrictionList
     plural: rolebindingrestrictions
     singular: rolebindingrestriction
+  scope: Namespaced
```

Comment:
please don't reshuffle the lines. This will fail the verify-crd script I expect. For bootstrapping run update-crds after your script.